nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -618,12 +618,16 @@ static int
 ble_svc_audio_bass_ctrl_point_write_access(struct ble_gatt_access_ctxt *ctxt, uint16_t conn_handle)
 {
     struct ble_svc_audio_bass_ctrl_point_handler *handler;
+	uint16_t mbuf_len = OS_MBUF_PKTLEN(ctxt->om);
+	uint8_t opcode;
+	uint8_t data[MYNEWT_VAL(BLE_SVC_AUDIO_BASS_METADATA_MAX_SZ)];
```

Comment:
please follow the coding style - code is not aligned

Reply:
Done